flask_restplus_patched/namespace.py

```diff
@@ -46,6 +46,12 @@ def resolve_object(self, object_arg_name, resolver):
         ...        # user is a User instance here
         """
         def decorator(func):
+            if isinstance(func, type):
```

Comment:
Sorry, I have just realized that we need to rename `func` to `func_or_class` here since we support both use-cases.